hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -536,6 +536,76 @@
       reseed: 1
       run_timeout_mins: 120
     }
+    {
+      name: rom_e2e_asm_init_test_unlocked0
+      uvm_test_seq: chip_sw_rom_e2e_asm_init_vseq
+      sw_images: [
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_a:1:signed:ot_flash_binary:fake_prod_key_0",
+        "//sw/device/silicon_creator/rom/e2e:otp_img_e2e_bootstrap_entry_test_unlocked0:4",
```

Comment:
the name of the testpoint is `rom_e2e_asm_init` but `sw_images` has `e2e_bootstrap_entry_test`. Is this intended?

Reply:
The `sw_images` key in dvsim serves 2 purposes - indicate the list of images to be built using bazel, and indicate what pre-built images are to be backdoor loaded into which memories in the chip. The `:4` flag at the end indicates that this image needs to be backdoor loaded into OTP. If you see the `chip_sim_cfg.hjson`, there is a note that explains this.

Reply:
Thanks, I was actually referring to the differences between target names: `asm_init` vs `bootstrap_entry`.

Reply:
yes, this was intentional, he bootstrap_entry OTP image is the same OTP image used for the asm_init test [on the FPGA](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/e2e/BUILD;drc=304038c0829380e628cc17f65e03c1998c1d6ea4;l=437), hence we reuse it here too.

Reply:
SG thanks for the clarification!